CONTRIBUTING.rst

```diff
@@ -112,39 +116,48 @@ These are the basic steps needed to start developing on Sphinx.
 
    For tips on working with the code, see the `Coding Guide`_.
 
-#. Test, test, test.  Possible steps:
+#. Test, test, test.
+
+   Testing is best done through `tox`, which provides a number of targets and
+   allows testing against multiple different Python environments:
+
+   * To list all possible targets::
+
+         tox -v
+
+   * To run unit tests for a specific Python version, such as 2.7::
```

Comment:
I prefer to use 3.6 for this example. 2.7 is too old...

Reply:
Done